gateway/ln-gateway/src/db.rs

```diff
@@ -154,12 +154,19 @@ async fn migrate_to_v1(dbtx: &mut DatabaseTransaction<'_>) -> Result<(), anyhow:
 }
 
 #[derive(Debug, Encodable, Decodable)]
-pub struct CreateInvoicePayloadKey(pub [u8; 32]);
+pub struct RegisteredIncomingContractKey(pub [u8; 32]);
+
+#[derive(Debug, Encodable, Decodable)]
+pub struct RegisteredIncomingContract {
+    pub federation_id: FederationId,
+    pub incoming_amount: u64,
```

Comment:
`IncomingContract` has the amount in the `Commitment`, is this just for convenience?

Reply:
Or does this value include fees?

Reply:
The two amounts differ by the fee. incoming_amount is lower and is the amount in the invoice.

Reply:
Can address in the future but a different name would help IMO

Reply:
incoming_htlc_amount?